# /database/msg/PotUpdate.msg

uint32[] update_list
uint32[] delete_list
